plugins/cloudbuild/README.md

```diff
@@ -0,0 +1,13 @@
+# cloudbuild
+
+Welcome to the cloudbuild plugin!
```

Comment:
This is Google Cloud Build, correct? I think that should be a bit more clear.

Reply:
I changed it to Google cloudbuild